assets/xml/objects/object_an1.xml

```diff
@@ -52,15 +52,15 @@
         <Texture Name="object_an1_Tex_0103A0" OutName="tex_0103A0" Format="ci8" Width="32" Height="32" Offset="0x103A0" />
         <!-- <Blob Name="object_an1_Blob_0107A0" Size="0xA40" Offset="0x107A0" /> -->
         <DList Name="object_an1_DL_0111E0" Offset="0x111E0" />
-        <DList Name="object_an1_DL_0111E8" Offset="0x111E8" />
+        <DList Name="object_an1_DL_0111E8" Offset="0x111E8" /> <!-- Original name is "an_obon_model" -->
```

Comment:
I thought we didn't have the original names of dlists

Reply:
Depends on what "type" of DList it is. For DLists that are part of a skeleton? Nope, they're completely gone. So in this object, for example, all of Anju's limb DLists are just part of `ange.cmb`; I'm 99% sure Grezzo came up with this name because it doesn't have the same prefix as her animations. However, DLists that *aren't* part of a skeleton usually *do* have names. Since Anju's tray and chopsticks aren't part of her skeleton, they are named individually; they even match the `an_whatever` naming scheme.

There are some exceptions to this, mainly around DLists that *weren't* part of the skeleton in MM64 but *are* part of the skeleton in MM3D. To give an example, MM64 Skull Kid has swappable hand DLists that he can use to pose his hands in certain ways. In MM3D, these are all just part of his skeleton, so we "lost" these names.

Does that make sense? It's a bit more of an art than a science, I'll admit, but we do have what are seemingly the original names of *some* DLists.

Reply:
Ohhh, very interesting. Yes it makes sense.
Thanks for the explanation

Reply:
Sometimes, we get objects like Anju's, where most of the things she "holds onto" in-game are actually separate models. It looks like only her mask was moved to her hand:
![image](https://user-images.githubusercontent.com/12245827/201526155-7434369a-cd0c-4a7f-a16f-2405798f1a69.png)

For other objects, like Human Link, basically *all* of his separate DLists were moved into his skeleton, depriving us of a ton of original names:
![image](https://user-images.githubusercontent.com/12245827/201526206-d4fc9b73-5e6a-426b-b596-2db9cf568676.png)